gigamon/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "041cf2fe-f391-4d8b-930c-b700c648c683",
+  "app_id": "gigamon",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": " Deep observability into all application traffic across cloud, virtual, and physical infrastructure",
+    "title": "Gigamon",
+    "media": [
```

Comment:
Please make sure to add all of the images here like you did with the first 3 as seen below

Reply:
@emarsha94 , is it mandatory to add all the images under manifest.json ? The other images under "images" folder has to be seen in another page /place, not at the main page of hublisting. Please suggest.

Reply:
followed up via email